dali/server/daserver.cpp

```diff
@@ -398,14 +398,17 @@ int main(int argc, char* argv[])
         if (confIFile->exists())
             serverConfig.setown(createPTreeFromXMLFile(DALICONF));
 
+#ifndef _CONTAINERIZED
         ILogMsgHandler * fileMsgHandler;
         {
             Owned<IComponentLogFileCreator> lf = createComponentLogFileCreator(serverConfig, "dali");
             lf->setLogDirSubdir("server");//add to tail of config log dir
             lf->setName("DaServer");//override default filename
             fileMsgHandler = lf->beginLogging();
         }
-
+#else
+        PROGLOG("Daserver NOT logging to local file!");
```

Comment:
all standard components should probably call same common helper routine e.g. startComponentLogging()

Reply:
I think jlog's MODULE_INIT is essentially doing everything the startComponentLogging() would do. What are you envisioning it would do?